style.css

```diff
@@ -102,92 +193,299 @@
   font-weight: 700;
-  font-size:16px;
+  font-size: 16px;
   margin-top: 30px;
 }
 
-.button button:hover{
-  color: #ff5438;
-  background-color: #423737;
-  background: rgb(255, 255, 255, 0.5);
+.seemore button {
+  background-color: #fff;
+  padding: 10px;
+  font-size: 20px;
+  border-radius: 10px;
+  color: #000;
+  border: solid 1px #ddd;
+  width: 100%;
+}
 
+.button button:hover {
+  color: #ff5438;
+  background-color: #3b3e45;
 }
 
-.cardz-sector{
+.cardz-sector {
   padding: 30% 5%;
   background-repeat: no-repeat;
   background-size: cover;
 }
 
-.cardz-sector h2{
+.cardz-sector h2 {
   margin-bottom: 24px;
   text-align: center;
-  font-family: 'Lato',sans-serif;
+  font-family: 'Lato', sans-serif;
 }
 
-.cards p{
+.cards p {
   color: #ff5438;
-  font-family: 'Lato',sans-serif;
+  font-family: 'Lato', sans-serif;
+}
+
+.cards h2 {
+  margin-bottom: 0;
 }
 
-.cards{
-  
+.text {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.cards {
   display: flex;
   padding: 5px 10px;
   gap: 24px;
   border-radius: 5px;
   background-repeat: no-repeat;
-
+  margin-bottom: 20px;
 }
 
-.cards hr{
+.cards hr {
   width: 20%;
 }
 
-.cards img{
+.cards img {
   width: 30%;
-}
\ No newline at end of file
+}
+
+.seemore {
+  display: flex;
+  justify-content: center;
+  padding: 0 20px;
+}
+
+/* desktop media query */
+@media (min-width: 769px) {
+  .hamburger,
+  div.rehearsal-space,
+  .seemore {
+    display: none;
+  }
+
+  .nav-box {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .grey-bar ul {
+    background-color: #333;
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding: 10px 8%;
+    gap: 20px;
+    color: #fff;
+    justify-content: flex-end;
+  }
+
+  .desktop-bar {
+    padding: 0 8%;
+    display: flex;
+    align-items: center;
+    gap: 30%;
+    width: 100%;
+  }
+
+  .dropdown-desktp {
+    display: flex;
+    gap: 80px;
+  }
+
+  .dropdown-desktp a {
+    text-decoration: none;
+    color: black;
+  }
+
+  .logo img {
+    margin-bottom: 0;
+  }
+
+  .whole-body1 {
+    height: 100%;
+    padding: 0 8% -20% 8%;
+  }
+
+  .div-1 {
+    font-size: 50px;
+    width: 50%;
+  }
+
+  .div-2 {
+    font-size: 30px;
+    width: 60%;
+  }
+
+  .div-3 {
+    font-size: 25px;
+    width: 65%;
+    margin-top: 20px;
+  }
+
+  .visit-time {
+    font-size: 30px;
+    width: 100%;
+  }
+
+  .services {
+    padding: 75px 0;
+  }
+
+  .offers {
+    display: flex;
+    flex-direction: row;
+    gap: 5px;
+    justify-content: center;
+  }
+
+  .offers div {
+    flex-direction: column;
+    width: 13%;
+    gap: 20px;
+    text-align: center;
+  }
+
+  .submit-b button {
+    background-color: transparent;
+    font-size: 20px;
+    border-bottom: solid white 2px;
+  }
+
+  .submit-b button:hover {
+    background-color: transparent;
+    border-bottom: solid #ff5438 2px;
+  }
+
+  .cardz-sector {
+    padding-top: 0;
+    padding-bottom: 1px;
+  }
```

Comment:
- [ ] Great work displaying the speakers' section beautifully. To make your design even nicer and create a good user experience let's add some more padding on the `top` and `bottom` of the section as demonstrated in the provided design.

<table>
  <tr>
    <td align="middle">Your design</td>
     <td align="middle">Provided design</td>
  </tr>
  <tr>
    <td align="top">
<img width="1440" alt="Screenshot 2023-04-24 at 22 44 25" src="https://user-images.githubusercontent.com/107192076/234133031-e0bbbca9-e996-448c-abdc-7252f4e72c7d.png">

</td>
    <td align="top">
<img width="1082" alt="Screenshot 2023-04-24 at 22 45 14" src="https://user-images.githubusercontent.com/107192076/234133067-9ed24b8a-1baf-4c15-abc2-e78bb121ad9a.png">

</td>
  </tr>
 </table>